### DISCUSSION TITLE
请问首页如何添加阅读更多

### DISCUSSION THREAD

**Discussion by lem0nb** [ppoffice/hexo-theme-icarus#884] [Q&A]
如题 感谢解答

**Comment by ppoffice** [comment]
@lem0nb 如果有时间的话，去看看文档：https://ppoffice.github.io/hexo-theme-icarus/uncategorized/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98/#%E5%86%85%E5%AE%B9

**Reply by lem0nb** [reply]
感激不尽
